.github/workflows/cut-rc.yml

```diff
@@ -0,0 +1,503 @@
+          The following Jira issues are still open for release ${{inputs.version}}:
+
+          $ISSUES
+
+          Please contact the assignees to clarify the status.
+          EOF
+            echo "::error::There are non-closed Jira issues for version ${{inputs.version}}."
+
+            curl -f -sSL --get --data-urlencode "jql=$JQL" \
+            -H "Authorization: Bearer $JIRA_TOKEN" \
+            -H "Accept: application/json" \
+            "https://issues.redhat.com/rest/api/2/search" \
+            | jq -r ".issues[] | .key" | while read -r KEY
+            do
+              curl -f -sSL -X POST \
+                -H "Authorization: Bearer $JIRA_TOKEN" \
+                --data '{"body": "Release ${{inputs.version}} is ongoing. Please update the status."}' \
+                -H "Content-Type: application/json" \
+                "https://issues.redhat.com/rest/api/2/issue/$KEY/comment"
+            done
+
+            exit 1
+          fi
+          echo "All issues for Jira release ${{inputs.version}} are closed." >> $GITHUB_STEP_SUMMARY
+
+  variables:
+    name: Setup variables
+    uses: stackrox/stackrox/.github/workflows/variables.yml@master
+    with:
+      version: ${{format('{0}{1}', github.event.milestone.title, github.event.inputs.version)}}
+
+  postpone-prs:
+    name: Postpone open PRs
+    needs: variables
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Check open PRs
+        id: check
+        run: |
+          PRs=$(gh pr list -s open \
+          --repo "${{github.repository}}" \
+          --search "milestone:${{needs.variables.outputs.milestone}}" \
+          --json number \
+          --jq length)
+          if [ "$PRs" -gt 0 ]
+          then
+            echo "::set-output name=open-issues::$PRs"
+          fi
+
+      - name: Create next milestone
+        if: github.event.inputs.dry-run != 'true' && steps.check.outputs.open-issues != ''
+        run: |
+          if ! http_code=$(gh api --silent -X POST \
+            "repos/${{github.repository}}/milestones" \
+            -f "title"="${{needs.variables.outputs.next-milestone}}" \
+            2>&1)
+          then
+            if grep "HTTP 422" <<< "$http_code"
+            then
+              echo "Milestone ${{needs.variables.outputs.next-milestone}} already exists." >> $GITHUB_STEP_SUMMARY
+            else
+              echo "::error::Couldn't create milestone ${{needs.variables.outputs.next-milestone}}: $http_code"
+              exit 1
+            fi
+          else
+            echo "Milestone ${{needs.variables.outputs.next-milestone}} has been created." >> $GITHUB_STEP_SUMMARY
+          fi
+
+      - name: Move open PRs
+        run: |
+          for PR in $(gh pr list -s open \
+            --repo "${{github.repository}}" \
+            --search "milestone:${{needs.variables.outputs.milestone}}" \
+            --json number \
+            --jq ".[] | .number")
```

Comment:
This may work fine, but it has some drawbacks.
I would suggest to capture the command output into an array and then iterate over that array in the for loop. This would allow us catch potential mistakes and handle properly edge-cases (e.g., empty values).